exonum-java-binding/launcher-plugin/setup.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+
+# Copyright 2019 The Exonum Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from distutils.core import setup
+from distutils.command.build_py import build_py
+from distutils.command.clean import clean
+from distutils.spawn import find_executable
+from distutils.debug import DEBUG
+
+import os
+import sys
+import subprocess
+
+protobuf_source = "exonum_java_runtime_plugin/proto/service_runtime.proto"
+
+if 'PROTOC' in os.environ and os.path.exists(os.environ['PROTOC']):
+    protoc = os.environ['PROTOC']
+else:
+    protoc = find_executable('protoc')
+
+
+def generate_proto(source):
```

Comment:
I very much would not like to have this code in the project which does things that a build system must do.

Reply:
Unfortunately, it is the only possible option here and now. The only project I found that does the same thing (https://github.com/pupssman/protobuf-setuptools) does not support integration into our own `setup.py`. It's weird, but I wasn't able to find a solution.

Reply:
Changed to maven configuration of the core module